mkdocs.yml

```diff
@@ -0,0 +1,29 @@
+site_name: KubeVirt User-Guide
+site_url: https://kubevirt.io/docs
+site_description: Documentation for KubeVirt
+docs_dir: docs
+site_dir: site
+
+plugins:
+  - search
+  - awesome-pages
+
+repo_url: https://github.com/kubevirt/user-guide/
+remote_branch: master
+
+markdown_extensions:
+    - toc:
+        permalink: true
+
+theme:
+  name: readthedocs
+  navigation_depth: 2
+  prev_next_buttons_location: bottom
+  include_search_page: true
+  search_index_only: false
+  titles_only: false
+  highlightjs: true
+  hljs_languages:
+    - yaml
+  icon:
+    logo: KubeVirt_logo_color.svg
```

Comment:
Where is this displayed?

And, shouldn't it be under `assets/` ?

Reply:
I think I was tinkering here and trying to get the home icon to change.  It's supposed to be favicon and change the icon displayed on the browser tab.

We can update the path.  See b6a9d568ca0